cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/templates/etl/common/Constants.java

```diff
@@ -44,4 +44,11 @@
   public static final class Transform {
     public static final String SPECIFICATIONS = "templates.etl.adapter.transform.specifications";
   }
+
+  /**
+   * Constants related to Realtime Adapter.
+   */
+  public static final class Realtime {
```

Comment:
Just notice this, so we are creating classes for each constants for just grouping? Could we just make it like flat Strings inside the Constants class but prefix it for each module?